app/controllers/proofs_controller.rb

```diff
@@ -42,79 +50,83 @@ def index
     @search_query = params[:search]
 
     @proofs = if @status.nil?
-                Proof.where.not(status: :spam).order(:manifestation_id)
+                Proof.where.not(status: :spam)
               else
-                Proof.where(status: @status).order('updated_at DESC')
+                Proof.where(status: @status)
```

Comment:
here, too, ordering was useful -- since this is usually used to review *resolved* proofs (to undo a mistaken resolution, or to double-chech something), in the vast majority of cases, the desired proofs are some of the latest-resolved ones.